duckduckgo/Android
# removing old sticky search code

Task/Issue URL: https://app.asana.com/0/488551667048375/1171986548569356

**Description**:
The Sticky Search Experiment has now finished, it's time cleanup and remove all the experiment code. 

**Steps to test this PR**:
1. Open the app
2. Navigate to Setting
3. Check that Sticky Search setting is no longer present
